packages/edit-site/src/components/global-styles/font-library-modal/context.js

```diff
@@ -196,31 +203,47 @@ function FontLibraryProvider( { children } ) {
 		setIsInstalling( true );
 		try {
 			// Prepare formData to install.
-			const formData = makeFormDataFromFontFamily( font );
+			const fontFamilyFormData = makeFontFamilyFormData( font );
+			const fontFacesFormData = makeFontFacesFormData( font );
+			const fontFamilyId = await fetchInstallFontFamily(
+				fontFamilyFormData
+			)
+				.then( ( response ) => response.id )
+				.catch( ( e ) => {
+					// eslint-disable-next-line no-console
+					console.error( e );
+					throw Error( 'Unable to install font family.' );
+				} );
+
 			// Install the fonts (upload the font files to the server and create the post in the database).
-			const response = await fetchInstallFont( formData );
-			const fontsInstalled = response?.successes || [];
-			// Get intersecting font faces between the fonts we tried to installed and the fonts that were installed
-			// (to avoid activating a non installed font).
-			const fontToBeActivated = getIntersectingFontFaces(
```

Comment:
I'm not sure what the purpose of this function is anymore. Is it still needed?

Reply:
The version in `trunk` returns all the installed font faces for a given font family. getIntersectingFontFaces gets only the ones that were sent in that request and were installed successfully and activates the,.

Reply:
Due to the fact that now we make a new request per font face it doesn't seems to be needed.